source/jormungandr/jormungandr/realtime_schedule/synthese_xml_reader.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2001-2016, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from aniso8601.time import parse_time
+import xml.etree.ElementTree as et
+import logging
+from jormungandr.interfaces.parsers import date_time_format
+from aniso8601 import parse_time
+
+
+class SynthesePassage(object):
+    def __init__(self):
+        self.date_time = None
+        self.real_time = None
+        self.waiting_time = None
```

Comment:
Is it necessary to read it? It seems redundant with `date_time` (so possibly incoherent and we would have to choose, so the sooner the better)